skfem/utils.py

```diff
@@ -597,6 +603,68 @@ def condense(A: spmatrix,
     return ret_value if len(ret_value) > 1 else ret_value[0]
 
 
+def mpc(A: spmatrix,
+        b: ndarray,
+        S: Optional[DofsCollection] = None,
+        M: Optional[DofsCollection] = None,
+        T: Optional[spmatrix] = None,
+        g: Optional[ndarray] = None) -> CondensedSystem:
+    """Apply a multipoint constraint on the linear system.
+
+    Parameters
+    ----------
+    A
+    b
+        The linear system to constrain.
+    S
+        The set of DOFs that are eliminated from the system.
+    M
+        The set of DOFs that are kept in the system.
+    T
+    g
+        The constraint is of the form `x[M] = T @ x[S] + g`.
```

Comment:
I''d guess it should be 
```suggestion
        The constraint is of the form `x[S] = T @ x[M] + g`.
```
as the slave DOFs are the restricted ones, right?

Reply:
Ah, yes. Of course.